.github/workflows/tr.unit.tests.yaml

```diff
@@ -24,13 +24,13 @@ env:
 on:
   push:
     paths:
-      - .github/actions/tr-tests/**
+      - .github/actions/tr-unit-tests/**
       - .github/workflows/tr.unit.tests.yaml
       - traffic_router/**
   create:
   pull_request:
     paths:
-      - .github/actions/tr-tests/**
+      - .github/actions/tr-unit-tests/**
```

Comment:
I'm hoping to get the *TR Tests* GitHub Action to run the TR integration tests soon, so IMO we should leave it at *tr-tests* for now.